externs/shaka/manifest_parser.js

```diff
@@ -120,6 +120,13 @@ shaka.extern.ManifestParser = class {
    * @exportDoc
    */
   setMediaElement(mediaElement) {}
+
+  /**
+   * provide CloseSegmentIndexRegister
+   * @param {shaka.util.CloseSegmentIndexRegister} register
+   * @exportDoc
+   */
+  setCloseSegmentIndexRegister(register) {}
```

Comment:
If we want to add new method to publicly exposed interface, it will need to be documented better.
Preferably though, we shouldn't add new method but try to look for other solutions (maybe callback in `shaka.extern.ManifestParser.PlayerInterface`?)

Reply:
I have update the implementation by using a new callback in `shaka.extern.ManifestParser.PlayerInterface`.